talktorials/5_compound_clustering/T5_compound_clustering.ipynb

```diff
@@ -142,7 +142,7 @@
        "        "
```

Comment:
<p>Apply the changes also here:</p><p> </p><p><code>for cutoff in numpy.arange(0., 1.0, 0.1):</code></p><p><code>&nbsp;&nbsp;&nbsp;clusters = ClusterFps(fingerprints, cutoff=cutoff)</code></p><p><code>&nbsp;&nbsp;&nbsp;fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(10, 4))</code></p><p><code>&nbsp;&nbsp;&nbsp;ax.plot(range(1, len(clusters)+1), [len(c) for c in clusters])</code></p><p><code>&nbsp;&nbsp;&nbsp;ax.set_xlim([1, len(clusters)])</code></p><p><code>&nbsp;&nbsp;&nbsp;ax.set_ylim([0, len(clusters[0])+1])</code></p><p><code>&nbsp;&nbsp;&nbsp;ax.set_xlabel('Cluster index', fontsize=20)</code></p><p><code>&nbsp;&nbsp;&nbsp;ax.set_ylabel('Number of molecules', fontsize=20)</code></p><p><code>&nbsp;&nbsp;&nbsp;ax.tick_params(labelsize=16)</code></p><p><code>&nbsp;&nbsp;&nbsp;plt.show()</code></p><p> </p><p>When we do the refactoring round of all notebooks, let's really enumerate though subplots here (and maybe use shared x- and y-axis).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>